api/apps/api/src/modules/projects/project-details.controller.ts

```diff
@@ -39,11 +41,15 @@ export class ProjectDetailsController {
     @Param('id') id: string,
     @Req() req: RequestWithAuthenticatedUser,
   ): Promise<ProjectResultSingular> {
-    return await this.projectSerializer.serialize(
-      await this.projectsService.findOne(id, {
-        authenticatedUser: req.user,
-      }),
-    );
+    const result = await this.projectsService.findOne(id, {
+      authenticatedUser: req.user,
+    });
+
+    if (isLeft(result)) {
```

Comment:
as it can return `undefined`, forbidden seems to be wrong.

Reply:
Changed into a switch handler (and I will probably dedicate some time to create a utils like @hotzevzl requested on a previous PR because this is going to be repeated A LOT after ACL introduction).

Also improved how the error was being handled on GET to avoid forbidden in case the project was deleted.